docs/source/_themes/sphinx_rtd_theme/static/css/theme.css

```diff
@@ -1,4 +1,4 @@
-﻿@charset "UTF-8";
```

Comment:
I'm in the process of fixing this stuff. @yhasan's original commit included a bunch of brocade/ipfabric stuff, and I'm in the process of unwinding it. Should be fixed in latest commit